.github/workflows/issue-bot.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 New Relic Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: issue-bot
```

Comment:
Isn't `issue-commenter` or `issue-comment` or `bot-responder` a better name for this workflow? `issue-bot` does not carry any meaning.

Reply:
I think this is a matter of preference. For me, it makes perfect sense- this workflow is for a bot that handles issues. We may elect to expand this functionality in the future beyond simple comments and if so, the name `commenter` or `responder` will be limiting.

Reply:
I don't have a very strong opinion on this so it's not a blocker for me. However when I look at the list of available workflows, their names should indicate what they do. To me `issue-bot` does not convey that message - `issue-bot` is who will execute the action but not what the action is. Judging by the name I don't know what this workflow will do. `issue-commenter` is very precise and covers what the workflow's action is supposed to do - comment on an issue. If we need to add a workflow that will do something else with the issue, say auto-close, I don't see us expanding `issue-bot` workflow with that extra functionality, but I'd rather expect a new workflow `issue-autocloser`. All in all, I think workflow name should describe what action the workflow is doing, not who is doing the action.

Reply:
After some thought I've changed the workflow name to `issue-comment` in this commit 8edd58cbdddfca77162afd28c99710ee9fe5ca26.

Reply:
Thank you!